docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -467,3 +525,134 @@ bin/yugabyted destroy --base_dir=/Users/username/yugabyte-2.3.3.0/data1
+
+bin/yugabyted start --advertise_address=<host-ip> --join=<ip-address-first-yugabyted-node> --cloud_location=aws.us-central.us-central-1a --fault_tolerance=region
+```
+
+After starting the Yugabyted processes in all the Nodes, configure the data placement constraint of the YugabyteDB cluster:
+
+```sh
+bin/yugabyted configure --fault_tolerance=region
+```
+
+This command will automatically determine the data placement constraint based on the `--cloud_location` of each node in the cluster. If there are 3 or more regions available in the cluster, configure command will be successful in configuring the cluster to survive at least 1 region failure. Otherwise it will print out the appropriate warning messages.
+
+Also the replication factor of the cluster will be default to `3`. Both of which can be manually set using `--data_placement_constraint` and `--rf` flags.
+
+Data placement constraint can be manually set with `--data_placement_constraint` flag which takes the comma separated value of `cloud.region.zone`. Example Usage:
+
+```sh
+bin/yugabyted configure --fault_tolerance=zone --data_placement_constraint=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a
+```
+
+Replication factor of the YugabyteDB cluster can be manually set with `--rf` flag. Example Usage:
+
+```sh
+./yugabyted configure --fault_tolerance=zone --data_placement_constraint=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a --rf=3
+```
+
+-----
+
+## YugabyteDB cluster upgrade
+
+In order to use the latest features of the database and also apply the latest security fixes, it's prudent to upgrade your YugabyteDB cluster to the latest available release as when they are available. This sections talks about doing a rolling upgrade of your YugabyteDB cluster using yugabyted. Upgrading a existing YugabyteDB cluster will involve the following steps -
+
+- Step 1: [Download](https://download.yugabyte.com/) the latest release of YugabyteDB and extract the release to home directory
+
+- Step 2: Stop the running YugabyteDB node using `yugabyted stop` command
+
+- Step 3: Start the new YugabyteDB process by executing the `yugabyted start` command from the `bin` directory. Use the previously configured `--base_dir` while restarting the YugabyteDB instance.
+
+Repeat the above steps on all the nodes of a YugabyteDB cluster, one node at a time.
+
+### Examples
+
+#### Steps for upgrading a YugabyteDB cluster from Single zone to Multi Zone configuration
+
+{{< note title="Note" >}}
+
+- Multi-zone, Multi-region deployment is supported in the latest [YugabyteDB 2.15.0.0](https://download.yugabyte.com/) release.
+
+{{< /note >}}
+
+These steps assume that you already have a running YugabyteDB cluster deployed using `yugabyted`.
+
+Step 1: Stop the first node by using `yugabyted stop` command
+
+```sh
+bin/yugabyted stop
+```
+
+Step 2: Download the [YugabyteDB 2.15.0.0](https://download.yugabyte.com/) release on the home directory
```

Comment:
Please don't mention specific versions unless they're _minimum versions_.

Also, who downloads anything to their home directory??
```suggestion
Step 2: Download the current [YugabyteDB preview release](https://download.yugabyte.com/).
```